examples/air-purifier-app/air-purifier-common/include/relative-humidity-sensor-manager.h

```diff
@@ -19,6 +19,9 @@
 #pragma once
 
 #include <app-common/zap-generated/attributes/Accessors.h>
+#include <protocols/interaction_model/StatusCode.h>
+
+using chip::Protocols::InteractionModel::Status;
```

Comment:
For the sake of namespace pollution, we souhld avoid `using` in .h.
Someday there might be other `Status`types in the SDK that might collide when including any of these .h 
(I suggest putting the user in the class definition below and ditching the `chip` since we will already be in the `chip` namespace at that point)

Reply:
This applies for all other .h in this PR.

Reply:
This is not using a namespace, simply a type.